event_handler.go

```diff
@@ -8,6 +8,16 @@ import (
 	"github.com/nlopes/slack"
 )
 
+func joinText(first string, second string, divider string) string {
```

Comment:
isn't this the same as `strings.Join` ?

Reply:
it's similar but not the same, using `strings.Join(["text", ""], "\n")` produces: `"text\n"` 
while with `joinText("text", "", "\n")` we only get `"text"`